doc/ADMIN.md

```diff
@@ -2,4 +2,4 @@
 
 During the installation app will prompt you to install itself as public or private, if you choose public then anyone (not just YunoHost users) would be able to access your Shaarli instance.
```

Comment:
This bit ain't really useful, YunoHost's explanation from the core should already be okay-ish (or to be improved for all apps, not just this one in particular ... but cf also the already existing `help` key in `manifest.toml`) AND .. ADMIN.md is meant to be displayed after install and meant to contain "administration documentation" so not really relevant to have this info which is relevant only prior install

```suggestion
```